manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.locale.ja-JP.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS1.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: JiroShimizu.Jw_cad
+PackageVersion: 8.25a
+PackageLocale: ja-JP
+Publisher: Jiro Shimizu & Yoshifumi Tanaka
```

```suggestion
Publisher: Jw_cad
```
This must match Control Panel.